app/services/katello/pulp3/repository.rb

```diff
@@ -386,6 +392,29 @@ def lookup_distributions(args)
         distributions_api.list(args).results
       end
 
+      def self.orphan_distributions(pulp3_repo, options)
```

Comment:
can 'options' be renamed to something more clear, like 'api_page_options' ?  Also, add a default value {}, so {} doesn't have to specified when used